src/edgeDebugAdapter.ts

```diff
@@ -230,7 +230,8 @@ export class EdgeDebugAdapter extends CoreDebugAdapter {
                 return properties;
             });
 
-            this._edgeProtocolVersion = await this._chromeConnection.version;
+            let protocolVersion : TargetVersions = await this._chromeConnection.version;
+            this._edgeProtocolVersion = (protocolVersion === undefined) ? undefined : protocolVersion.protocol;
```

Comment:
It might make more sense to initialize this to Version.unknownVersionVersion() when undefined, so we don't need to remember to check for undefined every time we use this. It might also make sense to  make that change in this._chromeConnection.version so all the callers can assume that the version is never undefined

Reply:
Yes, that makes sense. Changing now